source/appModules/miranda32.py

```diff
@@ -22,6 +22,8 @@
 from keyboardHandler import KeyboardInputGesture
 import watchdog
 
+category_title = "Miranda32"

```

Comment:
Idem for constant naming and translability.

Also, the application's name is probably "Miranda", not "Miranda32"; to be checked.

Reply:
It is Miranda NG

Reply:
This comment still stands: please change the name to "Miranda NG" as confirmed by @LeonarddeR.